src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -5,13 +5,31 @@
     <Description>Monitor.Tests Class library</Description>
     <PackageId>Monitor.Tests</PackageId>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
-    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452</TargetFrameworks>
+    <!-- TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks -->
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <!-- PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.0.0" / >
+	  <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.8" / >
+	  <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.2" / -->
```

Comment:
That version of resource manager comes from AzSdk.test.reference.props, which is including it unconditionally. I have no explicit reference to this package/dll.
I am going to need guidance here:
I can add an explicit reference to the package called: <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />

That does not exclude the old reference to 1.1.0-preview. The build contains two dlls:
Microsoft.Azure.Management.ResourceManager.dll
Microsoft.Azure.ResourceManager.dll

Internally, the dlls are similar, but the namespaces and files are different.